### ISSUE TITLE
Skin framework <link...> tag missing closing .../>

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#7]
Skin framework <link...> tag missing closing .../>
==>Add the closing '/' to the link tag.
